zipkin-autoconfigure/storage-elasticsearch-http/src/main/java/zipkin/autoconfigure/storage/elasticsearch/http/ZipkinElasticsearchHttpStorageProperties.java

```diff
@@ -13,12 +13,13 @@
  */
 package zipkin.autoconfigure.storage.elasticsearch.http;
 
-import java.io.Serializable;
-import java.util.List;
 import okhttp3.OkHttpClient;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import zipkin.storage.elasticsearch.http.ElasticsearchHttpStorage;
 
+import java.io.Serializable;
```

Comment:
pls undo import formatting change. might want to consider setting up the same code style as we use since it will eliminate this sort of stuff https://github.com/openzipkin/zipkin/blob/master/.github/CONTRIBUTING.md#contributing-to-zipkin